javascript/javascript1/week1/houseyPricey.js

```diff
@@ -0,0 +1,43 @@
+//Housey pricey (A house price estimator)
+
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+//Peter's house cost evaluation. 
+
+const width = 8;
+const depth = 10;
+const height = 10;
+const houseVolume = width * depth * height;
+
+const gardenArea = 100;
+
+let housePrice = (houseVolume * 2.5 * 1000) + (gardenArea * 300);
+let priceDifference = (2500000 - housePrice)
+
+if (housePrice < 2500000) {
+    console.log( "Peter's house actual price is " + housePrice + " but he is paying " + priceDifference + " more.");    
+} else {
+    console.log( "Peter's house actual price is " + housePrice + " but he is paying " + priceDifference + " less.");    
+}
```

Comment:
This `if-else` is very well designed! You actualy display the difference in an easily readable way! 🤓 

This whole code reads like a paragraph, which is how code should be written. Well done! 💯